basenode/src/main/java/io/coti/basenode/services/TrustChainConfirmationService.java

```diff
@@ -87,6 +91,8 @@ public List<TccInfo> getTrustChainConfirmedTransactions() {
                 monitorTotalTrustScore(transactionData);
             }
         }
+        topologicalOrderedGraph.stream().forEach(
+                transactionData -> transactionTrustChainTrustScoreMap.put(transactionData.getHash(), transactionData.getSenderTrustScore()));
```

Comment:
plz check if the hasId of the object Double is the same after being copied to the new map as it was in the original object